styleguide/source/assets/scss/03-organisms/_resource-tabs.scss

```diff
@@ -267,12 +267,23 @@ section#schedules {
   border-bottom: solid 1px $gray-50;
 }
 
-section#related_links ul {
-  list-style: none;
-
-  li {
+section#related_links {
+  ul {
     list-style: none;
-    margin: 0;
+
+    li {
+      list-style: none;
+      margin: 0;
+    }
+  }
+
+  .ama__membership ul {
+    list-style: initial;
+
+    li {
+      list-style: initial;
+      margin-left: 28px;
```

Comment:
please use gutters instead
margin-right: $gutter;

Remember to put padding: 0;

Reply:
Perhaps you can try of using list-style-position: inside instead of margins?